src/app/homepage.component.html

```diff
@@ -65,6 +65,13 @@ <h2 class="title_event">{{ "Title_event4" | translate }}</h2>
                             <p class="subtitle_event">{{ "Sub_event4" | translate }}</p>
                         </div>
                     </div>
+                    <div class="item">
```

Comment:
You add the item but you do not add the 'carrousel indicator' with it, can you do it please ?

N.B. in `src/app/homepage.component.html`, you have to add at line 38 something like : 
```
<li data-target="#carouselExampleIndicators" data-slide-to="4"></li>
```

Reply:
Merci